configs/bb.org-debian-stretch-iot-grove-kit-v4.14.conf

```diff
@@ -116,6 +116,7 @@ deb_additional_pkgs="	\
 	python3-pyaudio \
 	python3-tqdm \
 	portaudio19-dev \
+	python3-evdev \
```

Comment:
This package doesn't exist in Debian Stretch: https://packages.debian.org/buster/python3-evdev